apps/aering/src/aer_abi.erl

```diff
@@ -0,0 +1,41 @@
+%%%-------------------------------------------------------------------
+%%% @copyright (C) 2017, Aeternity Anstalt
+%%% @doc
+%%%     Encode and decode data and function calls according to
+%%%     Ring-AEVM-ABI.
+%%% @end
+%%% Created : 25 Jan 2018
+%%%
+%%%-------------------------------------------------------------------
+-module(aer_abi).
+-define(HASH_SIZE, 32).
+
+-export([create_calldata/3]).
+
+create_calldata(Contract, Function, Argument) ->
+    %% TODO: check that function exists in cotract.
+    FunctionHandle = encode_function(Contract, Function),
+    Call = {FunctionHandle, Argument},
+    {0, Data} = aer_data:to_binary(Call),
+    _ArgumentType = get_type(Argument),
+    %% TODO: Verify that the type matches the function signature.
+    Data.
+
+encode_function(_Contract, Function) ->
+     << <<X>> || X <- Function>>.
```

Comment:
I understand this is equivalent to `list_to_binary(Function)`?

Reply:
Almost. It will truncate values > 255 to 8 bits without crashing.